src/herder/test/HerderTests.cpp

```diff
@@ -1259,7 +1259,7 @@ testSCPDriver(uint32 protocolVersion, uint32_t maxTxSize, size_t expectedOps)
         // combineCandidates() and the one we compute at each step.
 
         std::vector<CandidateSpec> const specs{
-            {0, 1, 100, 10, make_optional<uint32>()},
+            {0, 1, 100, 10, std::optional<uint32>()},
```

Comment:
this change looks odd
* should change all to `std::make_optional`
* `std::make_optional<uint32>()` should just work

Reply:
(yes, it does)